compiler/passes/scopeResolve.cpp

```diff
@@ -1653,14 +1653,16 @@ Symbol* lookupAndCount(const char*           name,
       if (renameLocs.size() > 0) {
         // this can be the case when we resolved an urse through a public import
         // that renames the symbol
-        checkConflictingSymbols(symbols, name, context, true, renameLocs);
+        checkConflictingSymbols(symbols, name, context, true, renameLocs,
+                                reexportPts);
       }
       else {
-        checkConflictingSymbols(symbols, name, context, storeRenames, renameLocs);
+        checkConflictingSymbols(symbols, name, context, storeRenames,
+                                renameLocs, reexportPts);
       }
       retval = NULL;
     } else {
-      retval = symbols[0];
+      retval = symbols[0]
```

Comment:
That looks wrong, there should be a semi colon there

Reply:
Sigh. Yes, thanks.